### PR TITLE
feat: [OI-227] no idps placeholder

### PR DESCRIPTION
**Pull request by mmoio**
- adds a placeholder in case no idp is found
- localization strings updated
- refactoring

### CODE REVIEW

File: src/oneid/oneid-ecs-core/src/main/webui/src/pages/login/SpidModal.tsx [hunk 1]
```diff
@@ -43,47 +86,22 @@ const SpidModal = ({ openSpidModal, setOpenSpidModal, idpList }: Props) => {
...
         </Grid>
-      </Grid>
+      ) : (
+        <Typography
+          variant="caption-semibold"
+          color="textPrimary"
+          sx={{ textAlign: 'center' }}
+          fontSize={16}
+        >
+          Nessun idp trovato
```
Comment (sebbalex) [inline src/oneid/oneid-ecs-core/src/main/webui/src/pages/login/SpidModal.tsx:102]: shouldn't be the localized var here?

Comment (mmoio) [reply src/oneid/oneid-ecs-core/src/main/webui/src/pages/login/SpidModal.tsx]: Sorry, I've amended my last commit

**Comment by sebbalex** [comment]
Looks good to me! I'm just wondering if also `SpidSelect` should be affected by the same logic, @mmoio wdyt?

**Comment by mmoio** [comment]
> Looks good to me! I'm just wondering if also `SpidSelect` should be affected by the same logic, @mmoio wdyt?

Sure, do i use the same UI? Just a string alerting that no idps are found?

**Comment by sebbalex** [comment]
> > Looks good to me! I'm just wondering if also `SpidSelect` should be affected by the same logic, @mmoio wdyt?
> 
> Sure, do i use the same UI? Just a string alerting that no idps are found?

Yes sure!

**Review by sebbalex — APPROVED** [review]

**Pull request merged**